src/collective/fieldcollapsing/browser/querybuilder.py

```diff
@@ -40,13 +43,13 @@ def __init__(self, query={'collapse_on': None}):
         self._base_results = set()
         self.query = query
         self.collapse_on = self.query.get('collapse_on', None)
-        self.collapse_on_parent = self.collapse_on == 'getParent'
+        self.collapse_on_parent = self.collapse_on == 'ParentContainer'
```

Comment:
no dude.... why aren't you listening. It has to be something that can never be a valid metadata field name.

Reply:
Regardless of what we put, someone can easily use it as a metadata field. However, call it `__PARENT__` instead.